datafusion/core/tests/sqllogictests/src/main.rs

```diff
@@ -29,53 +29,18 @@ mod setup;
 mod utils;
 
 const TEST_DIRECTORY: &str = "tests/sqllogictests/test_files";
-const TEST_CATEGORIES: [TestCategory; 2] =
-    [TestCategory::Aggregate, TestCategory::ArrowTypeOf];
-
-pub enum TestCategory {
-    Aggregate,
-    ArrowTypeOf,
-}
-
-impl TestCategory {
-    fn as_str(&self) -> &'static str {
-        match self {
-            TestCategory::Aggregate => "Aggregate",
-            TestCategory::ArrowTypeOf => "ArrowTypeOf",
-        }
-    }
-
-    fn test_filename(&self) -> &'static str {
-        match self {
-            TestCategory::Aggregate => "aggregate.slt",
```

Comment:
Rather than hard coding this list, I propose instead listing the files in the directory instead

Reply:
You did what I want to do 🤣